README.md

```diff
@@ -80,8 +80,9 @@ spec:
 * `--metrics-path <path>`: The HTTP path where prometheus metrics will be exposed. Default is `/metrics`."
 
 * `--http-endpoint <endpoint>`: The TCP network address where the HTTP server for diagnostics, including CSI driver health check and metrics. The default is empty string, which means the server is disabled.
+Several klog arguments such as `-v <log level>` and `-logging-format <log format>` are supported.
 
-* All glog / klog arguments are supported, such as `-v <log level>` or `-alsologtostderr`.
+* Some klog arguments are supported, such as `-v <log level>` or `-logging-format <log format>`.
```

Comment:
I think it should be `--v` and `--logging-format`. Also `--v` might be coming from klog but `--logging-format` is coming from component base

Reply:
Thank you for reviewing it.
I fixed this description. Could you please review it again?